content/codesprint-2024/contributions.md

```diff
@@ -0,0 +1,163 @@
+[@jansule](https://github.com/jansule).
+
+### GeoStyler Mapbox Parser
+
+The [geostyler-mapbox-parser](https://github.com/geostyler/geostyler-mapbox-parser) now successfully reads colors as hex-codes
+as well as RGBA strings into the GeoStyler style. This works for both hardcoded color values and color values in (nested) expressions.
+Also the parser now supports the newly introduced `interpolate` expression.
+
+Kudos to
+[@jansule](https://github.com/jansule).
+
+### GeoStyler SLD Parser
+
+The usability of the [geostyler-sld-parser](https://github.com/geostyler/geostyler-sld-parser) was improved by automatically reading
+the version of a given SLD from the style itself. Through this, it is now possible to up-/downgrade SLDs while using only a
+single instance of the parser. Additionally, the geostyler-sld-parser now supports translating its error messages into multiple languages,
+which allows for easier communicating styling errors with the style editors.
+
+Kudos to
+[@ocruze](https://github.com/ocruze),
+[@pprev94](https://github.com/pprev94).
+
+### GeoStyler UI
+
+The [GeoStyler UI](https://github.com/geostyler/geostyler) was improved by adding an upload file button to the code editor.
+Additionaly, the expresions UI was extended with elements for the `case` and `step` function expressions.
+
+![GeoStyler UI Step Expression](/images/geostyler-codesprint-2024-expression-step.png)
+
+Kudos to
+[@kaivolland](https://github.com/kaivolland),
+[@pprev94](https://github.com/pprev94).
+
+### GeoStyler Visual Comparator
+
+A not-yet published tool for visually comparing different styles that were processed by GeoStyler was improved by adding
+additional styles to compare as well as minor UI tweaks.
+
+![GeoStyler Visual Comparator](/images/geostyler-codesprint-2024-comparator.png)
+
+Kudos to 
+[@jansule](https://github.com/jansule).
+
+### GeoStyler Homepage Design
+
+A new design concept for the GeoStyler Homepage was created that serves all static content at one place. As part of that,
+the homepage, tutorials, showcases, demo applications and more should all be directly accessible from
+the homepage. The new design is intended to better deliver the message of GeoStyler while still being easy to maintain.
+
+![GeoStyler Homepage Design](/images/geostyler-codesprint-2024-design.png)
+
+Kudos to
+Line,
+[@sarastreit](https://github.com/saraStreit),
+[@antonellostruzzolino](https://github.com/antonellostruzzolino),
+[@slafayIGN](https://github.com/slafayIGN),
+[@jansule](https://github.com/jansule).
+
+### ESModule Builds
+
+The GeoStyler packages were harmonized and now all provide ESModule builds in their distrubtions. We also moved from
+webpack to vite and vitest in order to simplify our tooling.
+
+Kudos to
+[@kaivolland](https://github.com/kaivolland),
+[@hwbllmnn](https://github.com/hwbllmnn).
+
+### GeoStyler REST
+
+The [GeoStyler REST API](https://github.com/geostyler/geostyler-rest) received some updates and now makes use
+of bun.
+
```

Comment:
```suggestion
![GeoStyler Rest](https://github.com/geostyler/geostyler-homepage/assets/1849416/df75b0b2-32ab-4642-a322-b1276048df4f)
```

![GeoStyler Rest](https://github.com/geostyler/geostyler-homepage/assets/1849416/df75b0b2-32ab-4642-a322-b1276048df4f)

Reply:
... you probably want to save this like the other pictures.